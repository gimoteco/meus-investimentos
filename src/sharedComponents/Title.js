import React from "react";
import { Helmet } from "react-helmet";
import { Heading } from "rebass/styled-components";

const TitleHeader = ({ children }) => {
  return (
    <Heading as="h1" mb={3} color="primary">
      {children}
    </Heading>
  );
};

function Title({ children }) {
  return (
    <TitleHeader>
      <Helmet>
        <title>{children}</title>
      </Helmet>
      {children}
    </TitleHeader>
  );
}

export default React.memo(Title);
