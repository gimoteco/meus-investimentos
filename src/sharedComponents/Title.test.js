import React from "react";
import Title from "./Title";

describe("<Title />", () => {
  it("should render like the snapshot", () => {
    expect(<Title>hi</Title>).toMatchSnapshot();
  });
});
