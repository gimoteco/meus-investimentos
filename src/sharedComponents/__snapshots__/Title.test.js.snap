// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> should render like the snapshot 1`] = `
<UNDEFINED>
  hi
</UNDEFINED>
`;
